packages/wallets/src/components/WalletListCardDropdown/WalletListCardDropdown.scss

```diff
@@ -1,11 +1,78 @@
-.wallets-list-card-dropdown {
-    &__item {
-        display: flex;
-        gap: 1.6rem;
+.wallets-listcard-dropdown {
+    position: relative;
+    cursor: pointer;
+
+    .wallets-textfield__field {
+        cursor: pointer;
+    }
+
+    &__list {
+        &-item {
+            display: flex;
+            gap: 1.6rem;
+        }
 
         &-content {
             display: flex;
             flex-direction: column;
         }
     }
+
+    &__button {
+        all: unset;
+        right: 1.6rem;
+        transform: rotate(0);
+        transform-origin: 50%;
+        transition: transform 0.2s cubic-bezier(0.25, 0.1, 0.25, 1);
+
+        &--active {
+            transform: rotate(180deg);
+        }
+    }
+
+    &__items {
+        position: absolute;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        z-index: 2;
+        border-radius: 0.4rem;
+        background: var(--system-light-8-primary-background, #fff);
+        box-shadow: 0 3.2rem 6.4rem 0 rgba(14, 14, 14, 0.14);
```

Comment:
Can you check the CSS variables if we have something similar

Reply:
Updated